### PR TITLE
Add SNIA talk as a post

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <36732377+glimchb@users.noreply.github.com>

### CODE REVIEW

File: content/posts/2023-01-25-snia.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "OPI Project at SNIA 2023"
+date: 2022-08-23T05:06:30-06:00
+---
+
+At the recent [SNIA](https://snia.org) event, a presentation on
+the Open Programmable Infrastructure Project was presented by Paul Pindell (F5),
+Boris Glimcher (Dell), Mark Sanders (Dell), Elad Blatt (Nvidia),
+Steven Royer (RedHat) and Dr. Joseph L. White (Dell). See below for the link to the recording.
```
Comment (seroyer) [inline content/posts/2023-01-25-snia.md:9]: Red Hat

**Review by seroyer — COMMENTED** [review]
Fix Red Hat, and then it looks good.

**Review by pdp2shirts — APPROVED** [review]
assuming that the YouTube link works, and the main page of the website autoupdates to showing the three most recent posts and the all posts button will show the page with the post that drops off the list then this looks good.

**Review by seroyer — APPROVED** [review]

**Comment by glimchb** [comment]
> assuming that the YouTube link works, and the main page of the website autoupdates to showing the three most recent posts and the all posts button will show the page with the post that drops off the list then this looks good.

yep, tested locally

**Pull request merged**